src/Microsoft.AspNetCore.Mvc.Razor/Microsoft.AspNetCore.Mvc.Razor.nuspec

```diff
@@ -0,0 +1,43 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>$id$</id>
+    <version>$version$</version>
+    <authors>$authors$</authors>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <licenseUrl>$licenseUrl$</licenseUrl>
+    <projectUrl>$projectUrl$</projectUrl>
+    <iconUrl>$iconUrl$</iconUrl>
+    <description>$description$</description>
+    <copyright>$copyright$</copyright>
+    <tags>$tags$</tags>
+    <serviceable>$serviceable$</serviceable>
+    <repository type="git" url="$repositoryUrl$" />
+    <dependencies>
+      <group targetFramework=".NETStandard2.0">
+        <dependency id="Microsoft.AspNetCore.Mvc.ViewFeatures" version="$version$" exclude="Build,Analyzers" />
+        <dependency id="Microsoft.AspNetCore.Mvc.Razor.Extensions" version="$MicrosoftAspNetCoreMvcRazorExtensionsPackageVersion$" exclude="Build,Analyzers" />
+        <dependency id="Microsoft.AspNetCore.Razor.Runtime" version="$MicrosoftAspNetCoreRazorRuntimePackageVersion$" exclude="Build,Analyzers" />
+        <dependency id="Microsoft.CodeAnalysis.CSharp" version="$MicrosoftCodeAnalysisCSharpPackageVersion$" exclude="Build,Analyzers" />
+        <dependency id="Microsoft.CodeAnalysis.Razor" version="$MicrosoftCodeAnalysisRazorPackageVersion$" exclude="Build,Analyzers" />
+        <dependency id="Microsoft.Extensions.Caching.Memory" version="$MicrosoftExtensionsCachingMemoryPackageVersion$" exclude="Build,Analyzers" />
+        <dependency id="Microsoft.Extensions.FileProviders.Composite" version="$MicrosoftExtensionsFileProvidersCompositePackageVersion$" exclude="Build,Analyzers" />
+      </group>
+      <group targetFramework=".NETFramework4.6.1">
+        <dependency id="Microsoft.AspNetCore.Mvc.ViewFeatures" version="$version$" exclude="Build,Analyzers" />
+        <dependency id="Microsoft.AspNetCore.Mvc.Razor.Extensions" version="$MicrosoftAspNetCoreMvcRazorExtensionsPackageVersion$" exclude="Build,Analyzers" />
+        <dependency id="Microsoft.AspNetCore.Razor.Runtime" version="$MicrosoftAspNetCoreRazorRuntimePackageVersion$" exclude="Build,Analyzers" />
+        <dependency id="Microsoft.CodeAnalysis.CSharp" version="$MicrosoftCodeAnalysisCSharpPackageVersion$" exclude="Build,Analyzers" />
+        <dependency id="Microsoft.CodeAnalysis.Razor" version="$MicrosoftCodeAnalysisRazorPackageVersion$" exclude="Build,Analyzers" />
+        <dependency id="Microsoft.Extensions.Caching.Memory" version="$MicrosoftExtensionsCachingMemoryPackageVersion$" exclude="Build,Analyzers" />
+        <dependency id="Microsoft.Extensions.FileProviders.Composite" version="$MicrosoftExtensionsFileProvidersCompositePackageVersion$" exclude="Build,Analyzers" />
+        <dependency id="Microsoft.DiaSymReader.Native" version="$MicrosoftDiaSymReaderNativePackageVersion$" exclude="Build,Analyzers" />
```

Comment:
Interesting. Does adding this as a PackageReference not work?

Reply:
We wanted to avoid cross-compiling the project, but have a framework specific dependency. 😿

Reply:
So, when restoring this on .NET Framework, does NuGet get confused and expect `lib/net461/*.dll`, or does it correctly pick `lib/netstandard2.0/` even though it's using `<group targetFramework=".NETFramework4.6.1">` dependencies?

Reply:
It does:

```json
"Microsoft.AspNetCore.Mvc.Razor/2.1.0-preview2-t000": {
        "type": "package",
        "dependencies": {
          "Microsoft.AspNetCore.Mvc.Razor.Extensions": "2.1.0-preview2-30355",
          "Microsoft.AspNetCore.Mvc.ViewFeatures": "2.1.0-preview2-t000",
          "Microsoft.AspNetCore.Razor.Runtime": "2.1.0-preview2-30355",
          "Microsoft.CodeAnalysis.CSharp": "2.6.1",
          "Microsoft.CodeAnalysis.Razor": "2.1.0-preview2-30355",
          "Microsoft.DiaSymReader.Native": "1.7.0",
          "Microsoft.Extensions.Caching.Memory": "2.1.0-preview2-30355",
          "Microsoft.Extensions.FileProviders.Composite": "2.1.0-preview2-30355"
        },
        "compile": {
          "lib/netstandard2.0/Microsoft.AspNetCore.Mvc.Razor.dll": {}
        },
        "runtime": {
          "lib/netstandard2.0/Microsoft.AspNetCore.Mvc.Razor.dll": {}
        }
      },
```

Reply:
Cool.